crime-evidence/src/main/java/uk/gov/justice/laa/crime/evidence/service/IncomeEvidenceValidationService.java

```diff
@@ -0,0 +1,52 @@
+package uk.gov.justice.laa.crime.evidence.service;
+
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.apache.commons.lang3.StringUtils;
+import org.springframework.stereotype.Service;
+import uk.gov.justice.laa.crime.evidence.staticdata.enums.OtherEvidenceTypes;
+
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+import java.time.ZoneId;
+import java.util.Date;
+
+@Slf4j
+@Service
+@RequiredArgsConstructor
+public class IncomeEvidenceValidationService {
+
+    public void checkEvidenceReceivedDate(Date incomeEvidenceReceivedDate, Date applicationReceivedDate) {
+        Date currentDate = Date.from(LocalDateTime.now().atZone(ZoneId.systemDefault()).toInstant());
```

Comment:
There's a method in the `crime-commons-classes` `DateUtil` that does this, probably better to reuse that.

Reply:
Updated